examples/ex1.cpp

```diff
@@ -80,6 +81,9 @@ int main(int argc, char *argv[])
    args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
                   "--no-visualization",
                   "Enable or disable GLVis visualization.");
+   args.AddOption(&use_ginkgo_solver, "--use_ginkgo", "--ginkgo", "-ginkgo",
+                  "-gko",
+                  "Solve using ginkgo.");
```

Comment:
For a Boolean variable, the four parameters after the first one should be:

- short-form enable flag,
- long-form enable flag,
- short-form disable flag,
- long-form disable flag.

With that in mind, I think the last two should be replaced with more appropriate strings.